cura/MultiplyObjectsJob.py

```diff
@@ -2,24 +2,15 @@
 # Cura is released under the terms of the LGPLv3 or higher.
```

Comment:
Maybe this file can be moved into the operations subfolder (or create a separate jobs folder or something)?